### PR TITLE
Add template for FBIG.

### PR DESCRIPTION
**Pull request by yyycggg**
Re: https://github.com/cocos/3d-tasks/issues/14293

### Changelog

* Add template for FBIG.

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by minggo — APPROVED** [review]

File: templates/fb-instant-games/fbapp-config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "instant_games": {
+    "orientation": "PORTRAIT",
+    "override_web_orientation": "PORTRAIT",
+    "platform_version": "RICH_GAMEPLAY",
+    "navigation_menu_version": "NAV_FLOATING",
+    "custom_update_templates": {
+      "play_turn": {
+        "example": "Edgar played their move"
+      }
+    }
```
Comment (PPpro) [inline templates/fb-instant-games/fbapp-config.json]: what's this `custom_update_templates` field for?
can we remove it ?

File: templates/fb-instant-games/vconsole.min.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
...ue:n})}return e}catch(e){return[]}}},{key:"clearCookieList",value:function(){if(document.cookie&&navigator.cookieEnabled){for(var e=this.getCookieList(),t=0;t<e.length;t++)document.cookie=e[t].name+"=;expires=Thu, 01 Jan 1970 00:00:00 GMT";this.renderStorage()}}},{key:"clearLocalStorageList",value:function(){if(window.localStorage)try{localStorage.clear(),this.renderStorage()}catch(e){alert("localStorage.clear() fail.")}}},{key:"clearSessionStorageList",value:function(){if(window.sessionStorage)try{sessionStorage.clear(),this.renderStorage()}catch(e){alert("sessionStorage.clear() fail.")}}}])&&d(o.prototype,n),c&&d(o,c),t;var o,n,c}(n.default);o.default=p,e.exports=t.default})?n.apply(t,r):n)||(e.exports=i)},function(e,t){e.exports='<div class="vc-table">\n  <div class="vc-log"></div>\n</div>'},function(e,t){e.exports='<div>\n  <dl class="vc-table-row">\n    <dd class="vc-table-col">Name</dd>\n    <dd class="vc-table-col vc-table-col-2">Value</dd>\n  </dl>\n  {{for (var i = 0; i < list.length; i++)}}\n  <dl class="vc-table-row">\n    <dd class="vc-table-col">{{list[i].name}}</dd>\n    <dd class="vc-table-col vc-table-col-2">{{list[i].value}}</dd>\n  </dl>\n  {{/for}}\n</div>'}])});
```
Comment (PPpro) [inline templates/fb-instant-games/vconsole.min.js:10]: we don't inctroduce vconsole by default.
the vconsole is an optional item on Editor build panel, please refer to the Web-Mobile platform

File: templates/fb-instant-games/fbapp-config.json (hunk 1 above)
Comment (yyycggg) [reply templates/fb-instant-games/fbapp-config.json]: OK, we can remove it.

File: templates/fb-instant-games/vconsole.min.js (hunk 2 above)
Comment (yyycggg) [reply templates/fb-instant-games/vconsole.min.js:10]: Now vconsole is an optional item on the Editor build panel. This is just a template file that is copied when the vconsole option on the Editor build panel is selected.

**Review by PPpro — APPROVED** [review]

**Pull request merged**